KarhooUISDK/Screens/PrebookConfirmationScreen/KarhooBookingConfirmationViewModel.swift

```diff
@@ -15,9 +15,11 @@ protocol BookingConfirmationViewModel {
     var vehicleImagePlaceholder: String { get set }
     var journeyDetails: JourneyDetails { get set }
     var quote: Quote { get set }
+    var trip: TripInfo? { get set }
     var loyaltyInfo: BookingConfirmationLoyaltyInfo { get set }
```

Comment:
I know that you added only one new property to this viewModel but I would like to know why all this variables are not private let? Shouldn't expose only values required by view like `printedPickUpAddressLine1`, `printedPickUpAddressLine2` etc and methods like `onAddToCalendar`